src/styles/main.scss

```diff
@@ -16,18 +16,18 @@
 @import "Question";
 @import "StudentQuestionContainer";
 
-body {background: grey; font-family: $font}
+body {
+  background: #d4d6d9;
```

Comment:
If you want a white background then just make it white

Reply:
Also, if you want a white background, you'll need to find all the places with white text and make them black, i.e dashboard

Reply:
I can't say I'm a huge fan of the card background being the same as the page background. Could we make the default card some shade of grey or keep the blue and just make a lighter blue hover state? It doesn't really matter what I just feel like they should be different.